control-plane/subcommand/server-acl-init/command.go

```diff
@@ -736,17 +736,17 @@ type gatewayRulesGenerator func(name, namespace string) (string, error)
 type ConfigureGatewayParams struct {
 	// GatewayType specifies whether it is an ingress or terminating gateway.
 	GatewayType string
-	//GatewayNames is the collection of gateways that have been specified.
+	// GatewayNames is the collection of gateways that have been specified.
 	GatewayNames []string
-	//AuthMethodName is the authmethod for which to register the binding rules and policies for the gateways
+	// AuthMethodName is the authmethod for which to register the binding rules and policies for the gateways
 	AuthMethodName string
-	//RuleGenerator is the function that supplies the rules that will be added to the policy.
+	// RuleGenerator is the function that supplies the rules that will be added to the policy.
 	RulesGenerator gatewayRulesGenerator
-	//ConsulDC is the name of the DC where the gateways will be registered
+	// ConsulDC is the name of the DC where the gateways will be registered
 	ConsulDC string
-	//PrimaryDC is the name of the Primary Data Center
+	// PrimaryDC is the name of the Primary Data Center
 	PrimaryDC string
-	//Primary specifies whether the ConsulDC is the Primary Data Center
+	// Primary specifies whether the ConsulDC is the Primary Data Center
 	Primary bool
 }
```

Comment:
Some of these comments might get flagged by the linter for not ending in a period.

Reply:
Thanks!  I was failing at linting.